src/Image.re

```diff
@@ -77,3 +77,31 @@ external make:
   ) =>
   React.element =
   "Image";
+
+type error;
+
+[@bs.module "react-native"] [@bs.scope "Image"]
+external getSize:
+  (
+    ~uri: string,
+    ~success: (~width: float, ~height: float) => unit,
+    ~failure: error => unit=?,
+    unit
+  ) =>
+  unit =
+  "getSize";
+
+type requestId;
+
+[@bs.module "react-native"] [@bs.scope "Image"]
+external prefetch: (~uri: string) => requestId = "prefetch";
+
+[@bs.module "react-native"] [@bs.scope "Image"]
+external abortPrefetch: requestId => unit = "abortPrefetch";
+
+[@bs.module "react-native"] [@bs.scope "Image"]
+external queryCache: (~urls: array(string)) => unit = "queryCache";
+
+[@bs.module "react-native"] [@bs.scope "Image"]
+external resolveAssetSource: Source.t => unit = "resolveAssetSource";
+external resolveAssetSourceInt: int => unit = "resolveAssetSource";
```

Comment:
These functions should actually return `Source.t`, not `unit`.

Reply:
We probably don't need `resolveAssetSource`, it is basically identity, right?

And `resolveAssetSourceInt` should probably be

```re
external resolveAssetSource: Packager.required => Source.t = "resolveAssetSource";
```

(and `Source.fromRequired` should be removed?)?